### ISSUE TITLE
[BUG] PersonalDNSFilter is stopped but notification shows Active

### ISSUE THREAD

**Issue by ricardogarceb** [IngoZenz/personaldnsfilter#302]
Hi there. This has happen me in 2 different devices a Samsung phone and a Samsung tablet.

I configured PerdonalDNSFilter in both to ignore battery optimizations and I also configure it as Always On VPN. Still, I found out that in both devices the VPN disconnects after leaving the devices.

![Screenshot_20240405-153552_One UI Home](https://github.com/IngoZenz/personaldnsfilter/assets/16104126/b1cc630b-2117-49b7-b82d-f7e8fb3dab4f)

But this is not the problem I'm reporting. The problem is that I know VPN was disconnected because there was an Always On VPN warning of it, but the app's notification shows it is active. Here the screenshot. 

If I open the app it was in fact not loaded, no the app's notification is not working properly, thus this bug report.

**Comment by IngoZenz** [comment]
Samsung is very tricky regarding battery optimization. Please check https://dontkillmyapp.com/samsung.
Regarding the app notification I'll have a look ...

**Comment by ricardogarceb** [comment]
Yeah I knew that website from when I need to configure Xiaomi or Oppo phones, which are the worst killing apps in the background. Everything is configured as recommended for Samsung devives in my case.

I just wanted to add another thing on this bug. I had the same situation again and I tried to start pDNSf using the app's notification, but clicking on pause/start did nothing, no response, or change, so I had to manually open the app for it to start VPN.

I saw there are some f.requests related to adding a quick settings tile for pDNSf , but I read in a closed issue this was solved by this notification. Is there any reason/difficulty on having a QS tile? In the past I used Blokada and found that the QS tile was the quickest way to enable/disable the VPN.

**Comment by IngoZenz** [comment]
regarding QS tile, personalDNSfilter is a one man show - there isn't a team behind like for Blokada. I usually implement those parts which I need myselfe. The QS tile just has no prio to me.

**Comment by ricardogarceb** [comment]
Sure, no problem. Just asking out of curiosity, I did not mean to "demand" this feature

**Comment by Qixot09** [comment]
I am getting the same problem with Nebulo dns and Orbot, they are disconnected after activation. Which advice could I follow?
